code/modules/mining/fulton.dm

```diff
@@ -114,6 +104,7 @@ GLOBAL_LIST_EMPTY(total_extraction_beacons)
 				L.SetUnconscious(0)
 				L.drowsyness = 0
 				L.SetSleeping(0)
+				L.say(pick(list("AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAHHHHHH!", "AAAAAAAAAAAUGH!", "EEEEEEEEEEEEEEEEEEEEEEEK!", "OH NOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOO!", "WHOAAAAAAAAAAAAAAAAAAAAAAAA!", "NOOOOOOOOOOOOOOOO!! HELP ME!!!")))
```

Comment:
I know the last one is a meme, but please no.

Reply:
I'm not sure why this part of the feature was removed, but this prevents fultons from being a completely silent way to yeet someone. I think someone removed it because "lol memes" but this + the forced awakening ensures you can't just parapen Fulton someone into space with absolutely no way for them to call for help.

Reply:
scream comes with a sound and text already, this is not needed imo.

Reply:
Incredibly based

Reply:
secret cockvore pr